src/components/Button/Button.type.ts

```
@@ -0,0 +1,16 @@
+import {Theme} from '@theme/theme.type';
+
+export type ButtonSize = 'small' | 'medium' | 'semiLarge' | 'large';
+export type ButtonVariants = 'primary' | 'secondary' | 'tertiary' | 'destructive' | 'loading' | 'kakao';
+
+export interface ButtonStyleProps {
+  variants?: ButtonVariants;
+  size?: ButtonSize;
+  theme?: Theme;
+}
+
+export interface ButtonCustomProps {}
```

Biome (1.9.4)

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)